### PR TITLE
Add more phpdoc

### PR DESCRIPTION
**Pull request by core23**
## Subject

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Added
- Add phpdoc for access mappings
```

### CODE REVIEW

File: src/Admin/AbstractAdminExtension.php [hunk 1]
```diff
@@ -79,6 +79,9 @@ public function getPersistentParameters(AdminInterface $admin)
         return [];
     }
 
+    /**
+     * @return array<string, string|string[]>
```
Comment (VincentLanglet) [inline src/Admin/AbstractAdminExtension.php:83]: Isn't this better on the Interface ?

Comment (core23) [reply src/Admin/AbstractAdminExtension.php:83]: This is not possible, yet. There is only a `@method` definition for this.

Comment (VincentLanglet) [reply src/Admin/AbstractAdminExtension.php:83]: Oh yes indeed, didn't see my code was on master ^^' 

Should we add the phpdoc to the comment on the Interface then ?
```
    /**
     * Return the controller access mapping.
     */
    // NEXT_MAJOR: Uncomment this method
    // public function getAccessMapping(AdminInterface $admin): array;
```

**Review by VincentLanglet — APPROVED** [review]

**Review by franmomu — APPROVED** [review]

**Comment by franmomu** [comment]
Thanks @core23!

**Pull request merged**